src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -40,6 +40,47 @@ public void execute_deleteValidName_success() {
         assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_deleteValidNameUnfilteredList_success() {
+
+        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+        //filter the typical person list in the addressbook to only display Alice
+        model.updateFilteredPersonList(person -> person.isSamePerson(new Person(new FriendName("Alice Pauline"))));
+        expectedModel.updateFilteredPersonList(person -> person.isSamePerson(new Person(new FriendName("Alice Pauline"))));
+
+        assert(model.getFilteredPersonList().size() == 1);
+        assert(expectedModel.getFilteredPersonList().size() == 1);
+
+        Person personToDelete = model.getAddressBook().getPersonList().get(3);
+
+        //ensures that person to delete is not in the filtered person list
+        assert(!personToDelete.hasSameName(new Person(new FriendName("Alice Pauline"))));
+
+        //deletes the person that is not in the filtered person list
+        DeleteCommand deleteCommand = new DeleteCommand(personToDelete.getName());
+
+        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_PERSON_SUCCESS, personToDelete);
+
+        expectedModel.deletePerson(personToDelete);
+
+        assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_deleteValidNameDifferentCase_success() {
+        Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
+        DeleteCommand deleteCommand = new DeleteCommand(new FriendName(personToDelete.getName().fullName.toLowerCase()));
```

Comment:
How do you ensure that this person doesn't already have their name in lower case?

Reply:
Because the first person in the typical person list is by default `Alice Pauline`, but yup I should add a new person and test on the person to ensure that this test does not hinge on the default values in the typical person list.